### PR TITLE
Added the N.E.R.D. (Nanotrasen Emergency Response Drone) team as an admin ERT.

### PR DESCRIPTION
**Pull request by KathyRyals**
## About The Pull Request

Added the ability to call a Nanotrasen Emergency Response Drone Team in the Create Antagonist.

## Why It's Good For The Game

More admin tools for fun stuff, easier to make drones for the admins as well.

## Changelog
:cl: Cimika / Kathy Ryals
admin: Added the ability to call a N.E.R.D. (Nanotrasen Emergency Response Drone) team in the create antagonist verb.
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
guaranteed drones are broken messes by now and people will make 50 issue reports when admins spawn these

**Comment by KathyRyals** [comment]
This is fair but so far when spawning them I haven't encountered any bugs.

**Comment by Fikou** [comment]
> easier to make drones for the admins as well.

lie clicking f7, fill and drone mobspawn is easier

**Comment by davethwave** [comment]
Perhaps, but there's a higher chance for spawned drone to mess around and do nothing useful if made like that. If spawned via an ERT it gives players in the drones a sense of actual responsibility and would cut down on those with no actual know how in station repair joining the role when it's needed.

File: code/modules/admin/verbs/one_click_antag.dm [hunk 1]
```diff
@@ -455,10 +455,37 @@
...
+
+	candidates = pollGhostCandidates("Do you wish to be considered for a nanotrasen emergency response drone?", "Drone")
+
+	if(length(candidates) == 0)
+		return FALSE
+
+	while(length(candidates) && teamsize)
+		chosen_candidate = pick(candidates)
+		candidates -= chosen_candidate
+		nerd = new /mob/living/simple_animal/drone(spawnpoint)
```
Comment (ATH1909) [inline code/modules/admin/verbs/one_click_antag.dm:484]: should probably give these drones custom laws

**Comment by ATH1909** [comment]
> guaranteed drones are broken messes by now and people will make 50 issue reports when admins spawn these

I mean, we have that ghost role space drone ruin that spawns pretty much every round, and we haven't received any reports about drones from that.

File: code/modules/admin/verbs/one_click_antag.dm (hunk 1 above)
Comment (KathyRyals) [reply code/modules/admin/verbs/one_click_antag.dm:484]: They kind of crawl out of the vents so I think standard laws fit fine.

**Review by Timberpoes — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/one_click_antag.dm [hunk 2]
```diff
@@ -24,6 +23,7 @@
 		<a href='?src=[REF(src)];[HrefToken()];makeAntag=centcom'>Make CentCom Response Team (Requires Ghosts)</a><br>
 		<a href='?src=[REF(src)];[HrefToken()];makeAntag=abductors'>Make Abductor Team (Requires Ghosts)</a><br>
 		<a href='?src=[REF(src)];[HrefToken()];makeAntag=revenant'>Make Revenant (Requires Ghost)</a><br>
+		<a href='?src=[REF(src)];[HrefToken()];makeAntag=nerd'>Make N.E.R.D.(Requires Ghost)</a><br>
```
[the remark this answers is not in the document]
Comment (KathyRyals) [reply code/modules/admin/verbs/one_click_antag.dm:26]: I can't believe you would do that to me.

**Review by Timberpoes — APPROVED** [review]

**Pull request merged**